mimesniff/mime-type-sniffing/resources/generate_scriptable_mime_type_tests.py

```diff
@@ -0,0 +1,145 @@
+
```

Comment:
Any reason for the empty blank lines at the top of these files?  Next file has two, too.

Reply:
I do this in scripts where I'm not adding a hashbang line.

Decided to add it here instead.